modules/installation/pages/client-requirements.adoc

```diff
@@ -17,24 +17,112 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 [cols="1,1,1,1", options="header"]
 .Supported Client Systems
 |===
-| Operating System | Architecture | Traditional Clients | Salt Clients
-| {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
-| {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
-| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 8 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sles} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
-| {rhel} 8 | {x86}_64                                   | icon:question[role="gray"]    | icon:question[role="gray"]
-| {rhel} 7 | {x86}_64                                   | icon:check[role="green"]      | icon:check[role="green"]
-| {rhel} 6 | {x86}, {x86}_64                            | icon:check[role="green"]      | icon:check[role="green"]
-| CentOS 8 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| CentOS 7 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| CentOS 6 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| {opensuse} Leap 15 | {x86}_64                       | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 20.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 18.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+
+| Operating System
+| Architecture
+| Traditional Clients
+| Salt Clients
+
+| {sle} 15
+| {x86}_64, {ppc}, {ibmz}, ARM
+| {check}
+| {check}
+
+| {sle} 12
+| {x86}_64, {ppc}, {ibmz}, ARM
+| {check}
+| {check}
+
+| {sle} 11
+| {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}
```

Comment:
What is {ipf}? ia64? This is out of regular support and we do not offer LTSS for it. So we also build no client tools for it. So this is not supported.

Reply:
ipf: Itanium

Reply:
Itanium is EOL. It does not have a LTSS Channel.